app/templates/download_framework_users.html

```diff
@@ -28,13 +28,13 @@
     with
     items = [
       {
-          "title": "Suppliers who’ve applied or started an application",
+          "title": "Users who are on the framework, have applied or started an application",
```

Comment:
This content reads a bit ambiguous to me. It reads like we're telling the user that users who are on the framework, have applied or started an application.

Does it mean:
`'Users who are on the framework, have applied, or have started an application'`
`'Users who are on the framework and have applied or started an application'`